### ISSUE TITLE
Floating button is placed on the card in Group expenses view as well

### ISSUE THREAD

**Issue by Ks-Weasley** [SKS2024SE/Track-Ur-Tab#8]
![image](https://github.com/user-attachments/assets/671f48c9-8dab-4134-8443-5e9c1d8498e9)
This is similar to the previous issue. Here, the floating button is placed with respect to the View container's height, I guess. @SandhiyaS24 what are your thoughts on this?

**Comment by SandhiyaS24** [comment]
@Ks-Weasley Yes it is, the floating button got wrapped inside the render cards, I have corrected the code for group expense. See the below image.
![WhatsApp Image 2024-11-01 at 16 43 20_c4c02147](https://github.com/user-attachments/assets/db422553-d2d6-4349-ad11-c46777f6973c)
